src/otx/algorithms/common/adapters/torch/dataloaders/samplers/otx_sampler.py

```diff
@@ -0,0 +1,98 @@
+"""OTX sampler."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import math
+
+from torch.utils.data import Dataset
+from torch.utils.data.sampler import Sampler
+
+from otx.algorithms.common.utils.logger import get_logger
+from otx.algorithms.common.utils.task_adapt import unwrap_dataset
+
+logger = get_logger()
+
+
+class OTXSampler(Sampler):  # pylint: disable=too-many-instance-attributes
```

Comment:
In my thought, `OTXSampler` is for adapting `repeat` values in sampler. So how about changing it's name to provide more detailed info? For example `RepeatSampler` can be one of candidates.

Reply:
I agree with you, this is the personal preference. Since we'll handle the 1 > repeats for the large dataset or we might add some sampling logic at the future, I want to represent this Sampler as a functionality of the OTX as like adaptive dataset control.